app/src/main/java/org/wikipedia/database/DatabaseTable.kt

```diff
@@ -85,10 +85,11 @@ abstract class DatabaseTable<T>(private val tableName: String, val baseContentUR
     }
 
     protected open fun onUpgradeSchema(db: SQLiteDatabase, fromVersion: Int, toVersion: Int) {}
-    private fun createTables(db: SQLiteDatabase) {
+
+    protected fun createTables(db: SQLiteDatabase) {
         L.i("Creating table=$tableName")
         val cols = getColumnsAdded(dBVersionIntroducedAt)
-        db.execSQL("CREATE TABLE " + tableName + " ( " + cols.joinToString(", ") + " )")
+        db.execSQL("CREATE TABLE IF NOT EXISTS " + tableName + " ( " + cols.joinToString(", ") + " )")
```

Comment:
I believe it's totally safe to add `IF NOT EXISTS` here. This also makes it symmetrical with `dropTable` which uses `IF EXISTS`.